mypy/errors.py

```diff
@@ -389,13 +393,13 @@ def new_messages(self) -> List[str]:
         return msgs
 
     def targets(self) -> Set[str]:
-        """Return a set of all targets that contain errors."""
+        """Return a set of all targets that contain errors including ignored ones."""
         # TODO: Make sure that either target is always defined or that not being defined
         #       is okay for fine-grained incremental checking.
         return set(info.target
                    for errs in self.error_info_map.values()
                    for info in errs
-                   if info.target)
+                   if info.target) | self.ignored_targets
```

Comment:
I share your concerns, but what are other options? Clearly we need to somehow modify how files with ignored errors are reprocessed, because some tests fail. Maybe we should just ignore ignores when calculating `targets()` by having `total_error_info_map`? Or will this be essentially the same?

I think I tried to generate more deps from invalid `MemberExpr`s, but IIRC this lead to lots of extra dependencies .

Reply:
I'm with @msullivan -- the performance impact is not acceptable, since this may turn `O(size of changed module)` incremental runs to `O(size of program)` runs. If there is no obvious way to get some small number of tests to pass without this, it's okay to skip those tests for now and create a follow-up issue with a more detailed discussion. Which test cases would be affected by removing this?

Reply:
Hm, it looks like it is only one test. I will thus revert this change and open an issue shortly.